maxim/models/query_builder.py

```
@@ -16,6 +16,10 @@ class QueryRule():
 
 
 class QueryBuilder:
+    """
+    This class represents a query builder. Users can use this class to build a query rule for fetching prompts, agents or workflow from Maxim server.
```

Pylint (3.3.7)

[convention] Line too long (149/100)

(C0301)